src/more-infos/more-info-climate.html

```diff
@@ -208,6 +238,20 @@
     }.bind(this), 500);
   },
 
+  computeTargetTempHidden: function (stateObj) {
+    return stateObj.attributes.operation_mode.includes('Off');
```

Comment:
I am very confused how this can be used to detect if we should hide target temperature ? Why not just check if it has a target temperature ?